etc/config.xml

```diff
@@ -54,6 +54,7 @@
                 <can_cancel>1</can_cancel>
                 <can_authorize_vault>1</can_authorize_vault>
                 <can_capture_vault>1</can_capture_vault>
+                <threeds_flow>native</threeds_flow>
```

Comment:
I see we're setting as default. Do we need a migration script to update old configs? That might be too much of a breaking change though

Reply:
As this field has been introduced with this PR, there is no need to update/migrate the config table. The default value will be obtained from this file in any case.

Reply:
Ah you're right, this is a new field